lib/prismic/tag_converter.rb

```diff
@@ -0,0 +1,113 @@
+module Prismic
+  class TagConverter
+    PRISMIC_FIELD = {
+      paragraph: 'p',
+      heading1: 'h1',
+      heading2: 'h2',
+      heading3: 'h3',
+      heading4: 'h4',
+      heading5: 'h5',
+      heading6: 'h6',
+      :'list-item' => 'li',
+      image: '<img src="%{src}" width="%{width}" height="%{height}" />',
+      :'o-list-item' => 'li'
+    }.stringify_keys.freeze
+
+    PRISMIC_FORMAT = {
+      strong: 'strong',
+      em: 'em',
+      hyperlink: '<a href="%{link}">%{text}</a>'
+    }.stringify_keys.freeze
+    attr_accessor :fragment
+    delegate :original_fragment, :field_type, :text, :text_format, to: :fragment
+    include ActiveModel::Model
+
+    # Transform Prismic fields into html tags. The possible values:
+    #  [
+    #   "paragraph",
+    #   "list-item",
+    #   "embed",
+    #   "image",
+    #   "o-list-item",
+    #   "heading2",
+    #   "heading3",
+    #   "heading4",
+    #   "heading1",
+    #   "heading6",
+    #   "heading5"
+    #  ]
+    #
+    def to_html
+      result = convert_special_formats
+
+      if field_type == 'embed'
+        original_fragment['data']['html']
+      elsif field_type == 'image'
+        image = original_fragment['data']['origin']
+        html_tag % {
+          src: original_fragment['data']['url'],
+          width: image['width'],
+          height: image['height']
+        }
+      else
+        "<#{html_tag}>#{result}</#{html_tag}>"
+      end
+    end
+
+    def html_tag
+      PRISMIC_FIELD[field_type].to_s
+    end
+
+    private
+
+    # Prismic formats. The "spans" node has this possible values:
```

Comment:
can you re-word this to `The "spans" node can have the following possible values:`